src/core/utils/default_nav_groups.ts

```diff
@@ -64,16 +64,16 @@ const defaultNavGroups = {
     }),
     order: 5000,
   },
-  analytics: {
-    id: 'analytics',
-    title: i18n.translate('core.ui.group.analytics.title', {
-      defaultMessage: 'Analytics',
+  essentials: {
+    id: 'essentials',
+    title: i18n.translate('core.ui.group.essential.title', {
+      defaultMessage: 'Essentials',
     }),
-    description: i18n.translate('core.ui.group.analytics.description', {
```

Comment:
We can not remove the `analytics` entry until we get https://github.com/opensearch-project/dashboards-observability/pull/2044 merged.